build-tools/api-xml-adjuster/Makefile

```diff
@@ -50,15 +50,9 @@ clean-api-xml:
 	rm -f $(CLASS_PARSE_XML) $(ANALYZED_XML)
 	$(XBUILD) /t:Clean
 
-$(CLASS_PARSE_XML): $(CLASS_PARSE) $(ANDROID_JAR) $(DOCS_DIR_CUR_LEVEL)
+$(CLASS_PARSE_XML): $(CLASS_PARSE) $(ANDROID_JAR) # $(DOCS_DIR_CUR_LEVEL)
 	mkdir -p $(CLASS_PARSE_XML_DIR)
-	if [ '$(LEVEL)' -eq 'O' ]; then \
-		$(RUN_CLASS_PARSE) $(ANDROID_JAR) -docspath=$(DOCS_DIR_CUR_LEVEL)/reference --docstype=droiddoc2 > $(CLASS_PARSE_XML) || rm -f $(CLASS_PARSE_XML) ;\
-	elif [ $(LEVEL) -le 25 ]; then \
-		$(RUN_CLASS_PARSE) $(ANDROID_JAR) -docspath=$(DOCS_DIR_CUR_LEVEL)/reference --docstype=droiddoc > $(CLASS_PARSE_XML) || rm -f $(CLASS_PARSE_XML) ;\
-	else \
-		$(RUN_CLASS_PARSE) $(ANDROID_JAR) -docspath=$(DOCS_DIR_CUR_LEVEL)/reference --docstype=droiddoc2 > $(CLASS_PARSE_XML) || rm -f $(CLASS_PARSE_XML) ;\
-	fi
+	$(RUN_CLASS_PARSE) $(ANDROID_JAR) -platform=$(LEVEL) -parameters-description=../../src/Mono.Android/Profiles/api-$(LEVEL).params.txt > $(CLASS_PARSE_XML) || rm -f $(CLASS_PARSE_XML)
```

Comment:
Should this use [`--parameter-names`](https://github.com/xamarin/java.interop/commit/25af6875722b5c27241b7df0b6d12d7b8802b5a1)?

Reply:
It turned out that your hack to detect documentation type is buggy. I'm going to create another PR to fix it.

Reply:
https://github.com/xamarin/java.interop/pull/212